core/src/main/java/org/stellar/anchor/sep31/Sep31Service.java

```diff
@@ -99,7 +99,7 @@ public Sep31PostTransactionResponse postTransaction(
     }
     validateRequiredFields(assetInfo.getCode(), request.getFields().getTransaction());
```

Comment:
WDYT if we also remove the arguments passing to `validateRequiredFields` if possible to make the style more consistent.  But feel free not to take care of this at this moment.

Reply:
This has been addressed, as well as adding a new call to `AssetService` to get `AssetInfo` with only the asset code.